### ISSUE TITLE
Issue with pyipmi.interfaces

### ISSUE THREAD

**Issue by atreyeeg** [kontron/python-ipmi#21]
I have installed the "pip install pyipmi"

[root@5d6ffa863468 ~]# python
Python 2.7.7 (default, Jul 21 2015, 10:02:08) 
[GCC 4.4.7 20120313 (Red Hat 4.4.7-11)] on linux2
Type "help", "copyright", "credits" or "license" for more information.
>>> import pyipmi
**>>> import pyipmi.interfaces
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ImportError: No module named interfaces**

**Comment by hthiery** [comment]
from the info you posted i cannot see whats going wrong. can you please check if the ipmitool.py tool is in your path?

try: which ipmitool.py

**Comment by atreyeeg** [comment]
no i dont see that is in my path : 

root@5d6ffa863468 main]# which pyipmi
/usr/bin/which: no pyipmi in (/usr/lib64/qt-3.3/bin:/usr/local/sbin:/usr/local/bin:/sbin:/bin:/usr/sbin:/usr/bin:/root/bin)
[root@5d6ffa863468 main]# 

this is the snippet of the pip install 👍 


==========
[root@5d6ffa863468 main]# pip install pyipmi
Collecting pyipmi
  Downloading http://artifactory-appliance.engba.veritas.com/artifactory/api/pypi/build-tools-pip-remote/packages/24/7e/afe2113670f10467a21dfd55793dee63cdbdbf2ef802fe898e64f4d987f4/pyipmi-0.11.0.tar.gz
Requirement already satisfied: pexpect in /usr/local/lib/python2.7/site-packages (from pyipmi)
Requirement already satisfied: ptyprocess>=0.5 in /usr/local/lib/python2.7/site-packages (from pexpect->pyipmi)
Installing collected packages: pyipmi
  Running setup.py install for pyipmi ... done
Successfully installed pyipmi-0.11.0
[root@5d6ffa863468 main]# 
=============

I dont see that the interfaces is installed in the site-packages:

[root@5d6ffa863468 main]# ls /usr/local/lib/python2.7/site-packages/pyipmi/
bmc.py       bootparam.py   chassis.py   data.pyc  event.pyc        freeipmi_pef.pyc  fw.pyc       __init__.pyc  mc.pyc   pet.pyc  sel.pyc     sol.pyc   watchdog.py
bmc.pyc      bootparam.pyc  chassis.pyc  dcmi.py   fabric.py        fru.py            info.py      lan.py        pef.py   sdr.py   server.py   tools     watchdog.pyc
bootdev.py   channel.py     commands     dcmi.pyc  fabric.pyc       fru.pyc           info.pyc     lan.pyc       pef.pyc  sdr.pyc  server.pyc  user.py
bootdev.pyc  channel.pyc    data.py      event.py  freeipmi_pef.py  fw.py             __init__.py  mc.py         pet.py   sel.py   sol.py      user.pyc
[root@5d6ffa863468 main]#

**Comment by hthiery** [comment]
you have installed pyipmi instead of python-ipmi package.... we had to name our package python-ipmi because pyipmi was already used! sorry for the confustion. Try to install python-ipmi.

**Comment by atreyeeg** [comment]
thanks it worked after installing "python-ipmi" .

So then do i need to install both the "pyipmi" and "python-ipmi" to make this module work ?

**Comment by hthiery** [comment]
No. You only need python-ipmi. There is also another project called pyipmi.

Am 08.02.2017 12:43 vorm. schrieb "Atreyee" <notifications@github.com>:

**Comment by atreyeeg** [comment]
thanks , but the example mentioned in this link  https://pypi.python.org/pypi/python-ipmi 

is importing the pyipmi , so is this example proper ?

If this is not proper can you please point me to the correct link ?

Thanks 
Atreyee

**Comment by hthiery** [comment]
this project is called python-ipmi also the pypi project's name is python-ipmi but the python module's name is pyipmi.

unfortunatly we named our python project pyipmi before release it to github and pypi. then we noticed there is already a project with the name pyipmi.

so you're fine by installing python-ipmi via pip from pypi and follow the instructions from the example...

i hope this brings some clarification
